products/cloudasset/api.yaml

```diff
@@ -105,6 +105,39 @@ objects:
                 required: true
                 description: |
                   Destination on Cloud Pubsub topic.
+      - !ruby/object:Api::Type::NestedObject
+        name: condition
+        required: false
+        description: |
+          A condition which determines whether an asset update should be published. If specified, an asset
+          will be returned only when the expression evaluates to true. When set, expression field
+          must be a valid CEL expression on a TemporalAsset with name temporal_asset. Example: a Feed with
+          expression "temporal_asset.deleted == true" will only publish Asset deletions. Other fields of
+          condition are optional.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: expression
+            required: true
+            description: |
+              Textual representation of an expression in Common Expression Language syntax.
+          - !ruby/object:Api::Type::String
+            name: title
+            required: false
+            description: |
+              Optional. Title for the expression, i.e. a short string describing its purpose.
+              This can be used e.g. in UIs which allow to enter the expression.
+          - !ruby/object:Api::Type::String
+            name: description
+            required: false
+            description: |
+              Optional. description of the expression. This is a longer text which describes the expression,
```

Comment:
Generated documentation for these will automatically have `Optional` as it can infer it from the `required` status. This can be removed

Reply:
Can be seen here: https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-4023-old..auto-pr-4023#diff-25d3ddf443da8ccac27f9619da229e09de79ccccb1ea527641eb27121fa9754bR188

Reply:
Removed "Optional" text from description.